Source/VisualStudioTools/Private/VisualStudioToolsCommandlet.cpp

```diff
@@ -114,39 +133,39 @@ struct AssetIndex
 
 	void ProcessBlueprint(const UBlueprintGeneratedClass* BPGC)
 	{
-		if(BPGC == nullptr)
+		if (BPGC == nullptr)
```

Comment:
I considered reverting these whitespace changes but they are fixing some bad formatting from the previous code, so would be good to include them too.